features/step_definitions/content.rb

```diff
@@ -55,3 +55,33 @@
   }
   expect(page).to have_xpath("//h1[contains(text(),'#{I18n.t *translations[text]}')]")
 end
+
+
+Then(/^there should be a back button on all pages except the start page$/) do
+  # These routes were excluded in accordance with our PO
+  excludedrouts = ['/', '/cable', '/tournaments/new', '/leagues/new', '/leagues/1/edit', '/tournaments/1/edit', '/events/1/edit', '/rankinglists/new', '/teams/new', '/teams/1/edit', '/matches/1/edit', '/my/users/sign_in', '/my/users/password/edit', '/my/users/cancel', '/my/users/sign_up', '/users/1/dashboard', '/users/1/edit', '/matches/1/add_game_result', '/matches/1/edit_results']
```

Comment:
multiple cases of "routs" instead of "routes" here. Also please rename your variables to be seperated by underscores like this: excluded_routes

Reply:
Fixed